include/hip/hcc_detail/math_functions.h

```diff
@@ -514,69 +514,69 @@ float __exp10f(float x) { return __ocml_exp10_f32(x); }
 __DEVICE__
 inline
 float __expf(float x) { return __ocml_exp_f32(x); }
-__DEVICE__
-inline
-float __fadd_rd(float x, float y) { return __ocml_add_rtp_f32(x, y); }
-__DEVICE__
-inline
-float __fadd_rn(float x, float y) { return __ocml_add_rte_f32(x, y); }
-__DEVICE__
-inline
-float __fadd_ru(float x, float y) { return __ocml_add_rtn_f32(x, y); }
-__DEVICE__
-inline
-float __fadd_rz(float x, float y) { return __ocml_add_rtz_f32(x, y); }
-__DEVICE__
-inline
-float __fdiv_rd(float x, float y) { return x / y; }
-__DEVICE__
-inline
-float __fdiv_rn(float x, float y) { return x / y; }
-__DEVICE__
-inline
-float __fdiv_ru(float x, float y) { return x / y; }
-__DEVICE__
-inline
-float __fdiv_rz(float x, float y) { return x / y; }
+// __DEVICE__
```

Comment:
Would it be better to use a guard here like #if defined OCML_BASIC_ROUNDED_OPERATIONS instead of comments?

Reply:
Okay I can use guards instead, its more flexible